scss/_utilities_vertical-spacing.scss

```diff
@@ -17,4 +17,42 @@
       }
     }
   }
+
+  .u-sv--shallow,
+  .u-sv--regular,
+  .u-sv--deep {
+    &::after {
+      @extend %u-vertical-spacer;
+    }
+  }
+
+  .u-sv--shallow {
+    &::after {
+      margin-top: calc($spv--strip-shallow / 2 - 1px); // subtracting the element height, which is set to 1px above.
```

Comment:
the "substracting element height…" is explained in first occurence in the file, so maybe we don't need to copy the comment every time?